modules/hostap/Kconfig

```diff
@@ -459,51 +462,9 @@ config WPS
 config WSC
 	bool
 
-config EAP_TLS
-	bool
-
 config IEEE8021X_EAPOL
 	bool
 
-config EAP_PEAP
-	bool
-
-config EAP_TTLS
-	bool
-
-config EAP_MD5
-	bool
-
-config EAP_MSCHAPv2
-	bool
-
-config EAP_LEAP
-	bool
-
-config EAP_PSK
-	bool
-
-config EAP_FAST
-	bool
-
-config EAP_PAX
-	bool
-
-config EAP_SAKE
-	bool
-
-config EAP_GPSK
-	bool
-
-config EAP_PWD
-	bool
-
-config EAP_EKE
-	bool
-
-config EAP_IKEv2
-	bool
-
```

Comment:
Note: EAP_IKEv2 and EAP_MSCHAPv2 are not duplicated. However, they are orphans.

Reply:
they are already in line 228 and line 204

Reply:
Indeed, I got trapped by the upper/lower case difference.